source/D2Common/src/D2Dungeon.cpp

```diff
@@ -1265,46 +1265,75 @@ void __stdcall DUNGEON_SaveKilledUnitGUID(D2RoomStrc* pRoom, D2UnitGUID nUnitGUI
 	}
 }
 
-//D2Common.0x6FD8D630 (#10111)
-void __stdcall DUNGEON_WorldToScreenCoords(int* pX, int* pY)
+//D2Common.0x6FD8D690 (#10107)
```

Comment:
I'd probably still prefer to keep things ordered by address instead of ordinal, but it doesn't matter too much

Reply:
I feel this order conveys better the intent of the devs, if it doesn't matter too much I'll keep it in this order :)